content/2017/02/09/generating-documentation-polyglot-projects.rst

```diff
@@ -0,0 +1,123 @@
+public: yes
+author: david
+tags: [
+  Docs,
+  Tools,
+  Javascript,
+  Python,
+  Sass
+  ]
+summary: |
+  Code documentation is ideally written as close to the actual code
+  as possible, but compiled into a comprehensive set of documentation
+  that includes code from all languages in use. Here's how we intend
+  to do that.
+
+
+Generating code documentation for polyglot projects
+===================================================
+
+At OddBird we believe that undocumented code is unfinished code.
+The act of documenting clarifies what we are building,
+and the resulting documentation guides consistency in how we do things
+in different parts of a project built by different team members
+at different times.
+
+In a typical project, one of our documentation deliverables is a
+living styleguide [link?]
```

Comment:
I have links about living styles guides but Im not sure that would be as helpful as a definition... 

A living style guide is a tool that is used not only as an inventory of the elements in a project, but also provides governance and usage information. What makes a style guide "living" is the ability to continually maintain the information in order to stay current with the site/application. At OddBird, we keep our style guides living by: 
• following a style guide *first* development process.
• writing our style guide documentation in our scss partials, minimizing the effort needed to stay current in a separate file or project.